bindings/python/test/test_builder.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from unittest import TestCase
+
+from pyarrow import Array, int32, int64
```

Comment:
Continuing my comment above, since we use pyarrow in the test suite, should we add it to `test_requires` or `install_requires`?

Reply:
Add a TODO do add these.

Reply:
If we add arrow to install_requires then we don't need `test_requires` at all.

Reply:
So far, this is correct. Eventually, we will likely need pandas for our tests but the package wouldn't really depend on it... We can add a test_requires for pandas when that happens.